tutorials/Getting_Started/README.md

```diff
@@ -1,63 +1,61 @@
 
 The following software has to be preinstalled:
    * JDK 11 or later
    * IntelliJ IDEA Community Edition or Ultimate Edition 20.2 or later (other editors could be used, but we assume you are using IntelliJ IDEA in this tutorial)
 
-## Creating a new project
+## Creating A New Project
 
-### New project wizard
+### New Project Wizard
 
-Kotlin support in IDEA IDE starting with the version 2020.3 comes with the new project wizard
-capable to create a Compose application automatically.
+Kotlin support in IDEA IDE starting with the version 2020.3, comes with the new project wizard which is
+capable of creating a Compose application automatically.
 
-Note that JDK must be at least JDK 11, and to use the native distribution
-packaging JDK 14 or later must be used.
+*Note that JDK must be at least JDK 11, and to use the native distribution
+packaging JDK 14 or later must be used.*
 
 ![Create new project 1](screen3.png)
 
 ![Create new project 2](screen4.png)
 
 ![Create new project 3](screen5.png)
 
-### Update the wizard plugin
+### Update The Wizard Plugin
 
-The Сompose plugin version used in the wizard above may be not the last. Update the version of the plugin to the latest available by editing the `build.gradle.kts` file, finding and updating the version information as shown below. In this example the latest version of the plugin was 0.3.0-build152 and a compatible version of kotlin was 1.4.30. For the latest versions, see the [latest versions](https://github.com/JetBrains/compose-jb/tags) site and the [Kotlin](https://kotlinlang.org/) site.
+The Сompose plugin version used in the wizard above may be not the latest. Update the version of the plugin to the latest available by editing the `build.gradle.kts` file, and then updating the version information as shown below. In this example the latest version of the plugin was 0.3.0-build152 and a compatible version of kotlin was 1.4.30. For the latest versions, see the [latest versions](https://github.com/JetBrains/compose-jb/tags) site and the [Kotlin](https://kotlinlang.org/) site.
 ```
 plugins {
     kotlin("jvm") version "1.4.30"
     id("org.jetbrains.compose") version "0.3.0"
 }
 ```
 
-### Create new Compose project without the wizard
+## Create A New Compose Project Without The Wizard
 
-It is also possible to create Compose project manually.
+It is also possible to create a Compose project manually.
 
 The recommended way of building Compose for Desktop projects is by using Gradle.
 JetBrains provides a simple way of building Compose for Desktop projects
 using a special Gradle plugin.
 
-One could clone an existing template for a [desktop](https://github.com/JetBrains/compose-jb/tree/master/templates/desktop-template) or
+One could clone an existing template for a [desktop program](https://github.com/JetBrains/compose-jb/tree/master/templates/desktop-template), or a
```

Comment:
It is possible to clone an existing template for a [desktop program](https://github.com/JetBrains/compose-jb/tree/master/templates/desktop-template), or a [multiplatform](https://github.com/JetBrains/compose-jb/tree/master/templates/multiplatform-template) application, or create it from scratch.

Reply:
changed